src/plots/gl3d/scene.js

```diff
@@ -58,10 +58,10 @@ function render(scene) {
     }
 
     function formatter(axisName, val) {
-        if(typeof val === 'string') return val;
-
         var axis = scene.fullSceneLayout[axisName];
-        return Axes.tickText(axis, axis.c2l(val), 'hover').text;
+        var calendar = trace[axis._id.charAt(0) + 'calendar'];
```

Comment:
don't we actually want to use the axis calendar for this? (which is what you get if you omit calendar as an arg to `d2l`)